docs/first-steps/introduction.md

```diff
@@ -0,0 +1,22 @@
+![binbash-logo](../assets/images/logos/binbash.png "Binbash"){: style="width:250px"}
+![binbash-leverage-tf](../assets/images/logos/binbash-leverage-terraform.png#right "Leverage"){: style="width:130px"}
+
+# Introduction
+
+The objective of this guide is to introduce the user to our [**Binbash Leverage Reference Architecture for AWS**](../how-it-works/) workflow through the complete deployment of a basic landing zone configuration.
+
+The Leverage landing zone is the smallest possible fully functional configuration. It lays out the base infrastructure required to manage the environment, billing and financial management, user and user access, security enforcement and shared services and resources. Always following the best practices layed out by the [AWS Well-Architected Framework](https://docs.aws.amazon.com/wellarchitected/latest/framework/welcome.html) to ensure quality and to provide a solid base to build upon. This is the starting point from which any Leverage user can and will develop all the features and capabilities they may require to satisfy their specific needs.
```

Comment:
I think this sentence needs to be revisited: "It lays out the base infrastructure required to manage the environment, billing and financial management, user and user access, security enforcement and shared services and resources." -- For instance, it says "user and user access".

Reply:
I would replace "user and user access, security enforcement" with "users and identity access management, security policies enforcement".

What do you think?